contributed_definitions/nyaml/NXdac.yaml

```diff
@@ -0,0 +1,10 @@
+category: base
+doc: |
+  Digital-to-analog converter component/integrated circuit.
+symbols:
+  doc: The symbols used in the schema to specify e.g. dimensions of arrays.
+NXdac:
```

Comment:
similarly to circuit_board, this can also be based on NXcircuit in the future

Reply:
Agreed